api/registration/models.py

```diff
@@ -77,4 +85,52 @@ def password_reset_token_created(sender, instance, reset_password_token, *args,
     subject = 'Reset your password'
     message = f'Dear {reset_password_token.user.full_name},\n\nClick on the link below to reset your password.\n{absolute_url}\n\nCLIMAN Team\n2023'
     msg = EmailMessage(subject, message, to=[reset_password_token.user.email])
-    msg.send()
\ No newline at end of file
+    msg.send()
+
+
+class Appointments(models.Model):
+    # The primary key for the Appointments model
+    # is a UUID field that is generated automatically.
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    # The foreign key field that references the User model
+    # and represents the doctor who will be seeing the patient.
+    doctor = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='doctor_appointments')
```

Comment:
This will cause the user model to be deleted with the appointment model if deleted
Replace cascade with protect